### PR TITLE
First iteration on FxOS prices form

### PR DESCRIPTION
**Pull request by comzeradd**
Not ready yet. Just to track the work on the UI.

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas r?

File: marketpulse/main/urls.py [hunk 1]
```diff
@@ -5,4 +5,6 @@
 urlpatterns = patterns(
     '{0}.main.views'.format(settings.PROJECT_MODULE),
     url(r'^$', 'home', name='home'),
+    url(r'^activities/$', 'activities', name='activities'),
+    url(r'^fxos_price/new/$', 'fxos_price_new', name='fxos_price_new'),
```
Comment (akatsoulas) [inline marketpulse/main/urls.py]: Maybe we should move each activity in a separate file eg activity_urls

File: marketpulse/main/templates/fxos_price_new.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    <div class="row">
+      <div class="col-md-6">
+
+        <fieldset>
+          <legend>Shop</legend>
+          <div class="form-group switch">
+            <label>Is this an online shop?</label>
+            <label class="pack-switch">
+              <input type="checkbox" id="onlineshop-check">
+              <span></span>
```
Comment (akatsoulas) [inline marketpulse/main/templates/fxos_price_new.html]: Is this needed somewhere or is it for future usage?

**Comment by akatsoulas** [comment]
Since most of this PR are static files, maybe @ppapadeas is more suitable than me to review the css files. Code-wise I believe we can merge it and fix any issues as we build upon this. Thoughts?

**Comment by ppapadeas** [comment]
r+ for the static files from me.

File: marketpulse/main/templates/fxos_price_new.html (hunk 2 above)
Comment (comzeradd) [inline marketpulse/main/templates/fxos_price_new.html]: It's a UI switch to reveal/hide relevant form fields (location, shop link, etc)

**Comment by comzeradd** [comment]
Did some minor changes based on comments.

**Pull request merged**